app/services/submit_reference.rb

```diff
@@ -18,11 +18,39 @@ def save!
     if @send_emails
       CandidateMailer.reference_received(reference).deliver_later
       RefereeMailer.reference_confirmation_email(application_form, reference).deliver_later
+      notify_provider_users
     end
   end
 
 private
 
+  def notify_provider_users
+    application_choices.each do |ac|
```

Comment:
this will need to be the application choice that the candidate has accepted an offer for. Otherwise if they have >1 application choice, the other providers will also be notified

Reply:
ahhh that makes this click now. thanks.

Reply:
can I rely on the `:awaiting_provider_decision status?` or would it be whether or not the `accepted_at` is present?
```ruby
enum status: {
  unsubmitted: 'unsubmitted',
  cancelled: 'cancelled',
  awaiting_provider_decision: 'awaiting_provider_decision',
  interviewing: 'interviewing',
  offer: 'offer',
  pending_conditions: 'pending_conditions',
  recruited: 'recruited',
  rejected: 'rejected',
  application_not_sent: 'application_not_sent',
  offer_withdrawn: 'offer_withdrawn',
  declined: 'declined',
  withdrawn: 'withdrawn',
  conditions_not_met: 'conditions_not_met',
  offer_deferred: 'offer_deferred',
}
```

Reply:
i think the relevant application choice will be in `offer` status